### ISSUE TITLE
handle dockerfile and helm Chart.yml package containers

### ISSUE THREAD

**Issue by dirkmueller** [openSUSE/cavil#44]
When a package only consists of a Dockerfile or a Chart.yaml (which is referring to a helm chart), then cavil fails the review wtih an error message. 

it would be good if we would continue to do source tarball evaluation (e.g. scan all the *tar* files in that package) instead, so that we can get something that is a valid report for legal.

**Comment by kraih** [comment]
Yes, that would be nice to support natively.

**Comment by kraih** [comment]
A few links to (reasonably diverse) samples would be very helpful to estimate the effort required to implement it.

**Comment by Vogtinator** [comment]
Most images are in `openSUSE:Factory` and are called `-image` or `-container`. Here's a list of all currently successfully building Dockerfile images in oS:F: https://build.opensuse.org/project/monitor/openSUSE:Factory?defaults=0&succeeded=1&arch_i586=1&&arch_x86_64=1&repo_containerfile=1

From a content point of view, there's no real difference to kiwi builds.

The included license statements and license files are solely for the content provided by the image description (including tarballs), but not any of the packages it installs during build or the base layers. Currently there is only a standard to express the license of the container as a whole (the `org.opencontainers.image.licenses` annotation).

**Comment by kraih** [comment]
The [devel:kubic:containers](https://build.opensuse.org/project/show/devel:kubic:containers) project contains lots of samples for Docker and Kiwi. Helm charts are more of a problem, since they don't have license metadata fields yet, just a bundled `LICENSE` file.

**Comment by Vogtinator** [comment]
> since they don't have license metadata fields yet, just a bundled LICENSE file.

Neither do `Dockerfile`s. For KIWI descriptions it's currently optional and most don't specify it.

**Comment by kraih** [comment]
> > since they don't have license metadata fields yet, just a bundled LICENSE file.
> 
> Neither do `Dockerfile`s. For KIWI descriptions it's currently optional and most don't specify it.

True, the convention to use `# SPDX-License-Identifier: BSD-3-Clause` comments does exist though. Even if in practice it is mostly missing.
